docs/CustomApp.md

```diff
@@ -19,8 +19,10 @@ Here is the default store creation for react-admin:
 
 ```js
 // in src/createAdminStore.js
-import { combineReducers, createStore, compose, applyMiddleware } from 'redux';
-import { routerMiddleware } from 'react-router-redux';
+import { applyMiddleware, combineReducers, compose, createStore } from 'redux';
+import { routerMiddleware, routerReducer } from 'react-router-redux';
+import formMiddleware from 'ra-core/lib/form/formMiddleware';
```

Comment:
There is indeed a problem, but you don't have the right fix. Importing modules from their path in the compiled code isn't supported in react-admin.

The problem is that the `formMiddelware` isn't exported by `ra-core`, and this should be fixed in `packages/ra-core/src/createAdminStore.ts`.

Reply:
Thanks for the clarification I'll try to make the needed changes ASAP. I'm not used to working with open source but I really like your project it helped me a lot.